java-frontend/pom.xml

```diff
@@ -68,6 +71,14 @@
 		<artifactId>logback-classic</artifactId>
       <scope>test</scope>
 	</dependency>
+  <dependency>
+    <groupId>commons-io</groupId>
+    <artifactId>commons-io</artifactId>
```

Comment:
@benzonico `commons-io` is used just in few places, where most likely it can be replaced by Java 8 APIs or Guava.